api/services/handle_event_service.py

```diff
@@ -23,6 +25,14 @@ def handle_event_service(self, form_data):
         if not system_phone_exists:
             return
 
+        user_details_exist = form_data.get("From")

```

Comment:
The name sounds like it will store a boolean value, which I believe is not correct.
Can we write it just  `user_details` or something similar?

Reply:
Sure, let me update it.